magento/magento2
# Fix syntax error in .editorconfig, closes #37394

### Description (*)

This fixes the issue #37394 for the synatx error in the editorconfig

### Fixed Issues (if relevant)

1. Fixes magento/magento2#37394

### Manual testing scenarios (*)

1. Open your editor, that supports the editorconfig.
2. View the composer or auth json file
3. See what the indent is for the composer and auth.json
4. See that configured size is now 4 instead of 2

### Contribution checklist (*)
